Lusas_oM/Versioning_63.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Namespace": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "Type": {
+    "ToNew": {
+
+    },
+    "ToOld": {
+    }
+  },
+  "Property": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "MessageForDeleted": {
+  },
+  "MessageForNoUpgrade": {
+    "BH.oM.Adapters.Lusas.LusasConfig()" : "This class has been renamed to LusasSettings and changed to inherit from AdaperSettings. Please replace this component with LusasSettings. For further assistance, please raise an issue on https://github.com/BHoM/Lusas_Toolkit/issues"
+  }
+}
```

Comment:
I don't think this is needed - because the object will upgrade from `LusasConfig` -> `LusasSettings`